src/main/java/eu/ha3/presencefootsteps/sound/Options.java

```diff
@@ -30,4 +31,5 @@ public float get(String option) {
     boolean containsKey(String option);
 
     float get(String option);
-}
+
+}
```

Comment:
Can you please go through and remove whitespace changes like these.

Reply:
I should've removed all the whitespaces in the new commit

Reply:
Thanks. This does look better. There's just a few whitespace changes that are still outstanding (this one and some indentations newline braces)

Reply:
I'm going to find the last whitespaces hoping I won't miss anything this time.